src/shared/component/post.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div
+    class="jumbotron jumbotron-fluid">
+    <div class="container">
+      <h1 class="display-4">{{news.title}}</h1>
+      <Modal v-if="read">
+        <div class="modal-content">
+          <p class="lead">{{ news.message }}</p>
+        </div>
+      </Modal>
+      <div class="action-buttons">
+        <h4>{{news.updatedAt}}</h4>
+        <button class="btn btn-primary" @click="readMore()">read</button>
+        <div class="action-button-left">
+          <p class="h3">
+            <i class="bi bi-trash bi-xl" @click="deletePost(news.id)"></i>
+          </p>
+          <p class="h3"><i class="bi bi-pencil-square" @click="editNewsFeed(news.id)"></i></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from 'vue'
+import RouteName from '@/enum/routes-name.enum'
+import { useRouter } from 'vue-router'
+import useDeleteById from '@/composables/feed/useDeleteById'
+import Modal from '@/shared/component/modal.vue'
+
+export default defineComponent({
+  name: "Post",
+  components: {
+    Modal,
+  },
+  props:{
+    id: {
```

Comment:
It looks like these props are properties of INewsfeed. How about using INewsfeed as a prop.

Reply:
used INewsfeed as prop